src/lib.rs

```diff
@@ -53,6 +55,10 @@ pub struct Renderer {
     shaders: FullShaders,
     blit: Option<BlitPipeline>,
     target: Option<TargetTexture>,
+    #[cfg(feature = "wgpu-profiler")]
+    profiler: GpuProfiler,
+    #[cfg(feature = "wgpu-profiler")]
+    pub profile_result: Option<Vec<wgpu_profiler::GpuTimerScopeResult>>,
```

Comment:
Should this be called `profile_results` as it can store multiple records?

Reply:
I don't think so; this is the results from a single frame.

The resulting value is only incidentally a vector